src/main/resources/static/scripts/interestMap.js

```diff
@@ -0,0 +1,34 @@
+const defaultView = [49.8037633, 15.4749126];
+const defaultZoom = 7;
+const mapUrl = 'https://tile.openstreetmap.org/{z}/{x}/{y}.png';
+const tileLayerOptions = {
+    maxZoom: 19,
+    attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+};
+
+let map = L.map('map');
+
+if (places.length === 0) {
+    map.setView(defaultView, defaultZoom);
+} else {
+    let markers = new L.featureGroup();
+
+    L.tileLayer(mapUrl, tileLayerOptions).addTo(map);
+
+    places.forEach(place => {
+        const popup = "<b><a href='/interests/1/places/" + place.id + "'>" + place.name + "</a></b>" +

```

Comment:
there is hardcoded 1 as the id. Is it correct?

Reply:
It's definitely not correct, but it doesn't actually play a role because of how we have the endpoints set, the id of interest plays no role here and any number works the same.